is_prime.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
Just add this docstring inside the function, so that these comments are automatically associated with the function they are about

https://peps.python.org/pep-0257/